### PR TITLE
Creating initial game board and adding piece images

### PR DESCRIPTION
**Pull request by austramarta**
- Creating logic for the chess board with canvas
- Adding assets folder with images

### CODE REVIEW

File: script_variant.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    fileName = fileName + pieceFileNames[piece.type]
+    img.setAttribute("src", fileName + ".png");
+
+}
+
+
+
+
+
```
Comment (klepzers) [inline script_variant.js]: Remove all but last newline!

File: script_variant.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+function clickOnPiece(e) {
+    console.log("Hello Chess2");
+    let mousePos = getMousePos(canvas, e);
+    console.log(mousePos);
+
+
+    console.log(Math.ceil(mousePos.x / squareSize));
+    console.log(Math.ceil(mousePos.y / squareSize));
+}
+
```
Comment (klepzers) [inline script_variant.js]: Remove duble empty lines

File: script_variant.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            } else {
+                ctx.fillStyle = "grey";
+            }
+            let xOffset = x * squareSize;
+            let yOffset = y * squareSize;
+            ctx.fillRect(xOffset, yOffset, squareSize, squareSize);
+        }
+    }
+}
+
```
Comment (klepzers) [inline script_variant.js]: Remove duplicate empty lines

File: logic.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+function getStartingPositions() {
+    return [
```
Comment (klepzers) [inline logic.js:3]: Add TODO TOMS comment!

**Review by klepzers — CHANGES_REQUESTED** [review]
Please remove .DS_Store (use local gitingore!!!)

Rename ambiguous script_variant.js to visuals.js or whatever meaningful!

File: logic.js (hunk 4 above)
Comment (klepzers) [reply logic.js:3]: And please, please(!!) - mark in comment all the types (including Knight => 'H')
👯

**Review by Toms-R — APPROVED** [review]

**Review by klepzers — CHANGES_REQUESTED** [review]

**Review by Toms-R — APPROVED** [review]

**Review by klepzers — APPROVED** [review]

**Comment by klepzers** [comment]
Super puper :)

**Pull request merged**